mite_browser/__init__.py

```diff
@@ -41,11 +41,11 @@ def url_builder(base_url, *args, **kwargs):
     return url
 
 
-def browser_decorator(separation=0):
+def browser_decorator(separation=0, embedded_res=False):
```

Comment:
```suggestion
def browser_decorator(separation=0, embedded_resources=False):
```

(I know it was like this in the previous code, but it's a bad name so we should avoid perpetuating it...)